client/qiskit_serverless/core/function.py

```diff
@@ -118,7 +118,7 @@ def run(self, **kwargs):
             config=config,
         )
 
-    def get_jobs(self):
```

Comment:
Something that we probably should do before merge is to add a deprecation warning here instead to remove it. Just to avoid a breaking change in the catalog until we release a new version.

Reply:
I can create an additional PR solving this.